paasta_tools/cli/cmds/status.py

```diff
@@ -1305,6 +1305,8 @@ def get_version_table_entry(
     if version_name_suffix is not None:
         version_name += f" ({version_name_suffix})"
     version_name = PaastaColors.blue(version_name)
+    if version.image_version is not None:
+        version_name += PaastaColors.blue(f" (image version: {version.image_version})")
```

Comment:
I think I'd prefer this to come before `version_name_suffix` as folks will probably be used to searching for `(old)` vs `(new)` at the *end* of the line during bounces, and that would keep sha+image_version together as the complete 'version' concept we're trying to push elsewhere...  
We can play with how we present this info together as we get closer, but let's at least keep the ordering consistent w/ status quo.

This also means you could do this addition to `version_name` before we colorify things, and won't have to re-insert multiple color codes for the same color :)